### PR TITLE
Test/block serializer args 1

### PR DESCRIPTION
**Pull request by sydhds**
* [ ] document all added functions
* [x] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

**Review by Eitu33 — COMMENTED** [review]
This is nice 👍

**Comment by Eitu33** [comment]
But is it really necessary for functions that only use a few arguments? I'm thinking about `BlockDeserializerArgs`, this would be nice but only for anything above the clippy limit imho.

**Comment by sydhds** [comment]
> But is it really necessary for functions that only use a few arguments? I'm thinking about `BlockDeserializerArgs`, this would be nice but only for anything above the clippy limit imho.

Indeed for struct with fee arguments, it's not very necessary but if we adopt this PR it would be nice to stay consistent in the code no?

**Comment by Eitu33** [comment]
I think what we should do is simply adopt the clippy rule, since the vast majority of function have few arguments it would add a lot of code, take time, and not bring much to the table. That said for those who are above the limit it's great.

**Comment by AurelienFT** [comment]
> * [ ]  document all added functions
> * [ ]  try in sandbox /simulation/labnet
> * [ ]  unit tests on the added/changed features
>   
>   * [ ]  make tests compile
>   * [ ]  make tests pass
> * [ ]  add logs allowing easy debugging in case the changes caused problems
> * [ ]  if the API has changed, update the API specification

Can you make all the checks here ? so that we are sure it doesn't break anything as we are close to the release date of the testnet we should be even more careful to try not include any new bug

**Review by Eitu33 — DISMISSED** [review]

**Comment by Eitu33** [comment]
Approving the code, did not test

**Comment by AurelienFT** [comment]
Postpone it to testnet_20 as it has been tested.

**Comment by sydhds** [comment]
Tested in sandbox mode and all good! @AurelienFT can you approve it?

File: massa-bootstrap/src/client_binder.rs [hunk 1]
```diff
@@ -19,33 +20,13 @@ use tokio::io::AsyncWriteExt;
 
 /// Bootstrap client binder
 pub struct BootstrapClientBinder {
-    max_bootstrap_message_size: u32,
+    // max_bootstrap_message_size: u32,
```
Comment (AurelienFT) [inline massa-bootstrap/src/client_binder.rs:23]: Can you remove this dead code ?

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**